### PR TITLE
Add "No recipes found!" text when recipes list is empty

### PR DESCRIPTION
**Pull request by HanHanzhi**
Screenshot 2022-03-26 at 10 28 47 PM
@GianniGabriel

### CODE REVIEW

File: src/components/layout/Search/SearchArea.module.css [hunk 1]
```diff
@@ -44,3 +44,12 @@
   padding: 7px 25px;
   border-radius: 10px;
 }
+
+.notFound {
+  color:#DE5151;
+  align-self: center;
```
Comment (GianniGabriel) [inline src/components/layout/Search/SearchArea.module.css:50]: Great job using `align-self` here! 👍

File: src/components/layout/Search/SearchArea.module.css [hunk 2]
```diff
@@ -44,3 +44,12 @@
...
+  font-size: 50px;
+  padding-top: 160px;
+  filter: drop-shadow(1px 1px 1px #413e3e);
```
Comment (GianniGabriel) [inline src/components/layout/Search/SearchArea.module.css]: I think the drop shadow is a bit too dark here. Try changing the `drop-shadow` color from `#413e3e` to `#9f9f9f`

**Review by GianniGabriel — CHANGES_REQUESTED** [review]
Nice work, one change requested before merging. Perform the requested changes, commit them to this branch, and @ me again so I can approve the new changes.

**Comment by HanHanzhi** [comment]
Screenshot 2022-03-26 at 10 51 09 PM
@GianniGabriel

**Review by GianniGabriel — APPROVED** [review]
Branch ready to merge into main. Really nice stuff resolving the merge conflicts. But it's a good idea to not resolve any merge conflicts until your PR has been approved.

**Comment by HanHanzhi** [comment]
@GianniGabriel Alright! will keep this in mind

**Pull request merged**